mod/photos.php

```diff
@@ -914,7 +914,9 @@ function photos_content(App $a)
 			'$submit' => DI::l10n()->t('Submit'),
 		]);
 
-		$usage_message = '';
+		$maximagesize_bytes = DI::config()->get('system', 'maximagesize');
+		$maximagesize_Mbytes = ($maximagesize_bytes / (10**6));
+		$usage_message = DI::l10n()->t('The maximum accepted image size is: ') . $maximagesize_Mbytes . ' MB';
```

Comment:
```suggestion
		$usage_message = DI::l10n()->t('The maximum accepted image size is %.3g MB', $maximagesize_Mbytes);
```

Reply:
That a great improvement, just thought about that I should add error handling for the division, if, for what ever reason, no useable number is returned by `DI::config()->get('system', 'maximagesize');`

Reply:
Yes, please check for the value first.